README.md

```diff
@@ -1,2 +1,5 @@
 # Dummy repo
 This repository is a blank repository where students can try different practices on GitHub.
+
+Students are able to view their grades both in an "all-courses" view by visiting student_allgrades.html.
```

Comment:
Naming is not consistent at all. Could you change "allgrades" filename to "allcourses" instead? This change needs to be reflected in this line.